portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -11,8 +11,8 @@
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Dif</h1>");
```

Comment:
This works if I use path /hello directly, but it's apparently not linked to from anywhere?